### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JacksonSchoe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,36 @@
...
+# Background
 This project is a mini survey monkey application. It allows users to 
 create and register surveys, and collect results from the surveys.
 
-To run, clone this repository and load into intelliJ. Running instructions
-will come later.
+A surveyor can create a survey with a list of Questions. Questions can be open-ended (text), asking for a number within a range, or asking to choose among many options.  Users fill out a survey that is a form generated based on the type of questions in the survey. Surveyor can close the survey whenever they want (thus not letting in new users to fill out the survey), and at that point a survey result is generated, compiling the answers: for open-ended questions, the answers are just listed as-is, for number questions a histogram of the answers is generated, for choice questions a pie chart is generated.
+
+# Installation Instructions
+  1. Create a folder for the project to be located in.
```
Comment (Fahid-Mannan) [inline README.md:14]: Not always required but thats fine

File: README.md [hunk 2]
```diff
@@ -1,10 +1,36 @@
...
+  
+  2. In your created folder, clone the repo from github using: 
+    SSH: git clone git@github.com:MichaelDysart/SYSC4806_Project.git
+    HTTPS: git clone https://github.com/MichaelDysart/SYSC4806_Project.git
+    
+  3. Open the cloned project folder in IntelliJ by running IntelliJ, clicking the "Open" button, and selecting the project          folder.
```
Comment (Fahid-Mannan) [inline README.md]: Should be the users choice as to what IDE they want to use

**Review by Fahid-Mannan — APPROVED** [review]
Good for now, maybe on another deliverable we can have images of on how to use our application and how to run tests

**Review by TaylorAbraham — APPROVED** [review]
Looks good to me!

**Pull request merged**